js/src/dropdown.js

```diff
@@ -526,7 +528,10 @@ class Dropdown extends BaseComponent {
       return
     }
 
-    Dropdown.selectMenuItem(Dropdown.getParentFromElement(this), event)
+    const context = Dropdown.getInstance(getToggleButton())
+    if (context) {
```

Comment:
Not sure if I really need to check for context here. I think this should be covered by checks for `!isActive` before, but just to be fail-safe? 🤔

@rohit2sharma95 WDYT?

Reply:
True,  the `_selectMenuItem` method will be called only when the instance for the dropdown exists. If it does not exist, the `_selectMenuItem` method with not be called because of the return statements.